contracts/0.4.24/Lido.sol

```diff
@@ -705,99 +616,41 @@ contract Lido is ILido, StETH, AragonApp {
     }
 
     /**
-    * @dev Emits {Transfer} and {TransferShares} events where `from` is 0 address. Indicates mint events.
-    */
+     * @dev Emits {Transfer} and {TransferShares} events where `from` is 0 address. Indicates mint events.
+     */
     function _emitTransferAfterMintingShares(address _to, uint256 _sharesAmount) internal {
         emit Transfer(address(0), _to, getPooledEthByShares(_sharesAmount));
         emit TransferShares(address(0), _to, _sharesAmount);
     }
 
-    /**
-    * @dev Deposits buffered eth to the DepositContract and assigns chunked deposits to node operators
-    */
-    function _depositBufferedEther(uint256 _maxDeposits) internal whenNotStopped {
-        uint256 buffered = _getBufferedEther();
-        if (buffered >= DEPOSIT_SIZE) {
-            uint256 unaccounted = _getUnaccountedEther();
-            uint256 numDeposits = buffered.div(DEPOSIT_SIZE);
-            _markAsUnbuffered(_ETH2Deposit(numDeposits < _maxDeposits ? numDeposits : _maxDeposits));
-            assert(_getUnaccountedEther() == unaccounted);
-        }
+    function getStakingRouter() public view returns (address) {
```

Comment:
return `IStakingRouter` and remove all checks like that:
```
require(stakingRouterAddress != address(0), "STAKING_ROUTER_ADDRESS_ZERO");
```

Reply:
fixed